rum-react-navigation-async/README.md

```diff
@@ -42,3 +42,16 @@ To try this project you need to create a folder called `config` in the project r
         "shopistApiHost": "shopist.io"
     }
     ```
+
+Moreover, since we have added the [`datadog-react-native-wizard`][1] you need to add the datadog-ci.json file in the project root containing:
```

Comment:
```suggestion
Since we have added the [`datadog-react-native-wizard`][1], you need to add the `datadog-ci.json` file in the project root that includes the following:
```